internal/build/docker_test.go

```diff
@@ -424,20 +426,43 @@ func (f *testFixture) startContainerWithOutput(ctx context.Context, ref string,
 
 func (f *testFixture) assertFilesInImageWithContents(ref string, contents []pathContent) {
 	ctx := context.Background()
-	var cmd strings.Builder
+	cId, err := f.b.startContainer(ctx, ref, &tiltd.Cmd{Argv: []string{"sh"}})
```

Comment:
huhhh i thought this func blocked until the container closed! how is the container still around for you to copy stuff?

Reply:
and actually on second thought, have you verified that this works even if the container has an entrypoint on it that causes it to exit immediately?

Reply:
hmmm...define "works"? like i can copy files out of the container, so it must work?